cmd/build.go

```diff
@@ -266,10 +266,8 @@ func dobuild(params buildParams, args []string) error {
 		return err
 	}
 
-	if bvc != nil || bsc != nil {
-		if !params.bundleMode {
-			return fmt.Errorf("enable bundle mode (ie. --bundle) to verify or sign bundle files or directories")
-		}
+	if bvc != nil || bsc != nil && !params.bundleMode {
+		return fmt.Errorf("enable bundle mode (ie. --bundle) to verify or sign bundle files or directories")
```

Comment:
[nit] This should work fine, but relies on operator precedence to keep the same meaning as before. I'd feel a little more comfortable with parens around the `||` conditions, since that's less likely to be missed at-a-glance. :sweat_smile: 
```suggestion
	if (bvc != nil || bsc != nil) && !params.bundleMode {
		return fmt.Errorf("enable bundle mode (ie. --bundle) to verify or sign bundle files or directories")
```